pkg/metadata/metadata.go

```diff
@@ -85,11 +84,17 @@ func (nodeManager *NodeInfoManager) NewNodeMetadata(logger *zap.Logger) (NodeMet
 	}
 
 	var workerID string
-	// In case of unmanaged cluster use label NodeInstanceIDLabel for workerID.
-	if os.Getenv(strings.ToUpper("IKS_ENABLED")) != "True" {
+
+	// If the cluster is satellite, the machin-type label equals to UPI
+	// For a satellite cluster, we fetch the workerID from instance ID labals, which is updated by vpc-node-label-updater (init container)
```

Comment:
*typo **labels**

Reply:
can you please change the tone of the speech for these comments.